R/FilteredData.R

```diff
@@ -451,8 +451,15 @@ FilteredData <- R6::R6Class( # nolint
     #' cat(shiny::isolate(datasets$get_formatted_filter_state()))
     #'
     get_formatted_filter_state = function() {
-      out <- c()
-      for (filtered_dataset in self$get_filtered_dataset()) out <- c(out, filtered_dataset$get_formatted_filter_state())
+      out <- Filter(
+        length,
+        sapply(
+          self$get_filtered_dataset(),
+          function(filtered_dataset) {
+            filtered_dataset$get_formatted_filter_state()
+          }
+        )
+      )
```

Comment:
Do you actually need to filter out here? If they do return NULL now

![image](https://user-images.githubusercontent.com/15201933/180960005-2afb48ff-54aa-49cf-a52a-bd154c1eb391.png)

Reply:
no, removed and updated `teal.reporter` PR accordingly.